components/advocates/MeetTheAdvocates.vue

```diff
@@ -8,6 +8,11 @@
       all over the world. If you are interested in getting involved with the
       quantum computing community, reach out to an advocate local to your area.
     </p>
+    <p class="copy__paragraph copy__paragraph_importance_support">
+      <strong>Note:</strong> you must have a <AppLink class="copy__link" :url="'https://ibm.co/joinqiskitslack'">
```

Comment:
You're using a fixed url, so use `url` instead of `:url`. Remeber that [`:` is the shorthand of `v-bind`](https://vuejs.org/v2/api/#v-bind) and [`v-bind` is used to bind HTML attribute to a JS expression](https://masteringjs.io/tutorials/vue/bind)